test-loop-tests/src/tests/catching_up.rs

```diff
@@ -425,99 +405,43 @@ fn test_all_chunks_accepted_common(
         .genesis(genesis)
         .clients(accounts)
         .epoch_config_store(epoch_config_store)
-        .config_modifier(move |config, _| {
-            let block_prod_time = Duration::milliseconds(block_prod_time as i64);
-            config.min_block_production_delay = block_prod_time;
-            config.max_block_production_delay = 3 * block_prod_time;
-            config.max_block_wait_delay = 3 * block_prod_time;
-        })
         .build()
         .warmup();
 
-    let verbose = false;
-
-    let seen_chunk_same_sender = Arc::new(RwLock::new(HashSet::<(AccountId, u64, ShardId)>::new()));
-    let requested = Arc::new(RwLock::new(HashSet::<(AccountId, Vec<u64>, ChunkHash)>::new()));
-    let responded = Arc::new(RwLock::new(HashSet::<(CryptoHash, Vec<u64>, ChunkHash)>::new()));
-
+    let seen_chunk_same_sender = Rc::new(RefCell::new(HashSet::<(AccountId, u64, ShardId)>::new()));
     for node_datas in &env.node_datas {
-        let sender_account_id = node_datas.account_id.clone();
-
         let seen_chunk_same_sender = seen_chunk_same_sender.clone();
-        let requested = requested.clone();
-        let responded = responded.clone();
-
         let peer_actor_handle = node_datas.peer_manager_sender.actor_handle();
         let peer_actor = env.test_loop.data.get_mut(&peer_actor_handle);
         peer_actor.register_override_handler(Box::new(move |msg| -> Option<NetworkRequests> {
-            let mut seen_chunk_same_sender = seen_chunk_same_sender.write();
-            let mut requested = requested.write();
-            let mut responded = responded.write();
             match msg {
                 NetworkRequests::PartialEncodedChunkMessage {
                     ref account_id,
                     ref partial_encoded_chunk,
                 } => {
                     let header = &partial_encoded_chunk.header;
-                    if seen_chunk_same_sender.contains(&(
+                    if seen_chunk_same_sender.borrow().contains(&(
                         account_id.clone(),
                         header.height_created(),
                         header.shard_id(),
                     )) {
                         println!("=== SAME CHUNK AGAIN!");
                         assert!(false);
                     };
-                    seen_chunk_same_sender.insert((
+                    seen_chunk_same_sender.borrow_mut().insert((
                         account_id.clone(),
                         header.height_created(),
                         header.shard_id(),
                     ));
                 }
-                NetworkRequests::PartialEncodedChunkRequest { ref request, .. } => {
-                    if verbose {
```

Comment:
verbose is set as false, remove dead code

Reply:
I think the author left it in to aid debugging - still, given they are not around it's fair game to remove it :)